src/com/palmergames/bukkit/towny/utils/TownPeacefulnessUtil.java

```diff
@@ -89,7 +97,7 @@ public static void punishPeacefulPlayersInActiveSiegeZones() {
 					continue;
 
 				//Dont apply if player has the immunity perm
-				if (TownyUniverse.getInstance().getPermissionSource().testPermission(player, PermissionNodes.TOWNY_SIEGE_WAR_IMMUNE_TO_WAR_NAUSEA.getNode()))
+				if (TownyUniverse.getInstance().getPermissionSource().testPermission(player, SiegeWarPermissionNodes.TOWNY_SIEGE_WAR_IMMUNE_TO_WAR_NAUSEA.getNode()))
```

Comment:
> If `TownPeacefulnessUtil` is going to stay in Towny, shouldn't the associated permission stay under Towny as well?

\[See previous [comment] on pre-[force push] version of code.\]

[comment]: https://github.com/TownyAdvanced/Towny/pull/4472#discussion_r531119425
[force push]: https://github.com/TownyAdvanced/Towny/compare/6cec7955c7b06dcf3df81d3cf249a2f1aac0c66c..8ff4c358df50cb3e830c397645ebf322db484380

Reply:
https://github.com/TownyAdvanced/Towny/pull/4472#discussion_r531167716

Reply:
Resolved in [original thread] for this change.

[original thread]: https://github.com/TownyAdvanced/Towny/pull/4472#discussion_r531119425